identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -490,6 +490,10 @@ hypr.generic.error=Authentication failed. Please try again later.
 hypr.auth.start=Authentication in progress. Please wait...
 hypr.invalid.email=Please enter a valid email address as the username.
 
+# UAE Pass
+error.uaepass.title=Failed Authentication with UAE PASS
```

Comment:
Do we need to show the end users the authenticator details (eg: UAE PASS) in the error message?

Reply:
This error is unique to UAE PASS authenticator, so it requires a unique error message